src/CONST.ts

```diff
@@ -984,6 +984,10 @@ const CONST = {
             GOLD: 'GOLD',
             SILVER: 'SILVER',
         },
+        STATEMENT_WEB_MESSAGE_TYPE: {
```

Comment:
I guess it could be just `WEB_MESSAGE_TYPE`.
```suggestion
        WEB_MESSAGE_TYPE: {
```